src/promise-katas-1.js

```diff
@@ -33,72 +33,128 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
 // 'resolve' Promise callback function in order to resolve successfully
 
-const returnPromise = () => { };
+const returnPromise = () => { 
+  return new Promise((resolve) => {
+    resolve();
+  });
+};
 
 // 2 Create a function that returns a promise which returns 10 if it resolves succesfully
 // Example:
 // returnTen() resolves to the value 10
 
-const returnTen = () => { };
+const returnTen = () => { 
+  return new Promise((resolve) => {
+    resolve(10);
+  });
+};
 
 // 3 Create a function that returns a promise which returns 'string' if it resolves succesfully
 // Example:
 // returnString() resolves to the value 'string'
 
-const returnString = () => { };
+const returnString = () => { 
+  return new Promise((resolve) => {
+    resolve('string');
+  })
+};
 
 // 4 Create a function that returns a promise which returns an object with a key of name and value of 'Bob' if it resolves succesfully
 // Example:
 // returnBob() resolves to the value { name: 'Bob' }
 
-const returnBob = () => { };
+const returnBob = () => { 
+  return new Promise((resolve) => {
+    resolve({ name: 'Bob' });
+  });
+};
 
 // 5 Create a function that returns a promise which returns an array with 4 strings of 'eggs', 'apples', 'milk' and 'bread' if it resolves succesfully
 // Example:
 // returnList() resolves to the value ['eggs', 'apples', 'milk', 'bread']
 
-const returnList = () => { };
+const returnList = () => { 
+  return new Promise((resolve) => {
+    resolve(['eggs', 'apples', 'milk', 'bread']);
+  });
+};
 
 // 6 Create a function that returns a promise which returns "An error occurred" if it rejects
 // Example:
 // anError() rejects to "An error occurred"
 
-const anError = () => { };
+const anError = () => { 
+  return new Promise((_, reject) => {
+    reject("An error occurred");
+  });
+};
 
 // 7 Create a function that returns a promise which returns 666 if it rejects
 // Example:
 // theNumberOfTheBeast() rejects to the value 666
 
-const theNumberOfTheBeast = () => { };
+const theNumberOfTheBeast = () => { 
+  return new Promise((_, reject) => {
+    reject(666);
+  });
+};
 
 // 8 Create a function that returns a promise which returns an object with a key of error and value of 500 if it rejects
 // Example:
 // internalServerError() rejects to { error: 500 }
 
-const internalServerError = () => { };
+const internalServerError = () => { 
+  return new Promise((_, reject) => {
+    reject({ error: 500 });
+  });
+};
 
 // 9 Create a function that returns a promise which resolves to "happy" if the parameter is greater than
 // or equal to 1 and rejects to "sad" if the value is less than 1
 // Example:
 // happySad(1) resolves to "happy"
 // happySad(0) rejects to "sad"
 
-const happySad = () => { };
+const happySad = (num) => { 
+  return new Promise((resolve, reject) => {
+    
+    if (num >= 1) {
+    resolve("happy");
+    } else {
+      reject("sad");
+    };
+  });
```

Comment:
You dont need the extra clear line here, and the contents of the if / else blocks should be indented by 2 spaces so its easy to see what code relates to what condition:
```
  return new Promise((resolve, reject) => {
    if (num >= 1) {
      resolve("happy");
    } else {
      reject("sad");
    };
  });
```